fixtures/peer-esbuild-gzip/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "peer-esbuild-non-gzip",
```

Comment:
I think this line should be renamed, too. As far as I can see on my small phone screen, this applies to every package.json that was moved, but where the package was not renamed. Doesn't make a difference, but it'd be more correct to rename them

Reply:
I just removed most of `package.name` since many of them was outdated baba6aa